ricecooker/utils/downloader.py

```diff
@@ -193,13 +193,7 @@ def download_assets(selector, attr, url_middleware=None,
     def js_content_middleware(content, url, **kwargs):
         if js_middleware:
             content = js_middleware(content, url, **kwargs)
-
-        # Polyfill localStorage and document.cookie as iframes can't access
-        # them
-        return (content
-            .replace("localStorage", "_localStorage")
-            .replace('document.cookie.split', '"".split')
-            .replace('document.cookie', 'window._document_cookie'))
```

Comment:
Can we separate this change out? I think we need to figure out how to handle the `document.cookie.split` change in a way that doesn't break old Kolibris.